### ISSUE TITLE
condition checks in the store

### ISSUE THREAD

**Issue by maxbachmann** [project-alice-assistant/ProjectAlice#163]
the interface store does not check all conditions before showing the skills.
```
 [SkillManager] Skill "Speller" does not comply to "asrArbitraryCapture" condition, required "True"
```
when it does not comply to a condition it should not even appear in the store

**Comment by Psychokiller1888** [comment]
No, it does not and it's intended, because these are settings you can change. Maybe displaying the conditions with a check mark?

**Comment by maxbachmann** [comment]
yeah and when it fails maybe show the reason in the web interface. Right now he will see that it failed and then has to check the logs

**Comment by maxbachmann** [comment]
and he should not be able to install -> maybe show them and hide the install button when it is not possible to install due to the settings
